index.yaml

```diff
@@ -26151,4 +26120,4 @@ entries:
     urls:
     - https://github.com/newrelic/helm-charts/releases/download/synthetics-minion-1.0.17/synthetics-minion-1.0.17.tgz
     version: 1.0.17
-generated: "2024-01-23T11:27:52.524988816Z"
+generated: "2024-01-23T05:55:18.144829537Z"
```

Comment:
I would not revert the time here since it can create issues (helm repo update does not work properly).
Instead, I would update it with a new time (2024-01-24T11:27:52.524988816Z)